libdnf/dnf-repo.c

```diff
@@ -214,20 +212,27 @@ dnf_repo_get_packages(DnfRepo *repo)
 }
 
 /**
- * dnf_repo_get_public_key:
+ * dnf_repo_get_public_keys:
  * @repo: a #DnfRepo instance.
  *
  * Gets the public key location.
  *
- * Returns: the repo public key location, e.g. "/var/cache/PackageKit/metadata/fedora/repomd.pub"
+ * Returns: (transfer full)(array zero-terminated=1): The repo public key location, e.g. "/var/cache/PackageKit/metadata/fedora/repomd.pub"
  *
- * Since: 0.7.0
+ * Since: 0.8.2
  **/
-const gchar *
-dnf_repo_get_public_key(DnfRepo *repo)
```

Comment:
Theoretically this is public API, but in practice neither PK nor rpm-ostree call it (and why would they?).  It's not part of the manual python bindings, and dnf still has separate keyring code.